src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/controller/AgreementController.java

```diff
@@ -51,6 +73,39 @@ public AgreementDetail getAgreement(@PathVariable(value = "ca-number") String ca
       throw new ResourceNotFoundException(
           String.format("Agreement number '%s' not found", caNumber));
     }
+    try {
+      URL url = new URL( wordpressURL + "wp-json/ccs/v1/frameworks/" + caNumber);
+      HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+      connection.setRequestMethod("GET");
+      connection.setRequestProperty("Accept", "application/json");
+
+      if (connection.getResponseCode() != 200) {
+    	  rollbar.error(String.format("Wordpress API returned %s, using database for summary and endDate" ,connection.getResponseCode()));
+      }else {
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(connection.getInputStream(), "UTF-8"));
+        JSONTokener tokener = new JSONTokener(bufferedReader);
+        JSONObject jsonObject = new JSONObject(tokener);
+
+        if (!StringUtils.isEmpty((String) jsonObject.get("summary"))) {
```

Comment:
Added a check to fall back using the database value in case the field was not populated on WordPress (aka its empty)